d3a_interface/utils.py

```diff
@@ -151,6 +151,10 @@ def key_in_dict_and_not_none(d, key):
     return key in d and d[key] is not None
 
 
+def key_in_dict_and_is_true(d, key):
+    return key in d and d[key] and d[key] is True
```

Comment:
```suggestion
    return key in d and d[key] is True
```

Reply:
upsi, thanks!

Reply:
To be sure: this is checking just for the `True` values, right? It needs to fail if the boolean value evaluates to true, but it's not the `True` object?

Reply:
More details about the question I asked: [PEP8 - Programming recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)
```
# Wrong:
if greeting is True:
```